c2cgeoportal/lib/email_.py

```diff
@@ -34,13 +34,23 @@
 from email.Utils import formatdate
 
 
-def send_email(from_addr, to_addrs, body, subject, smtp_server):  # pragma: no cover
+def send_email(from_addr, to_addrs, body, subject, smtp):  # pragma: no cover
     msg = MIMEMultipart()
     msg["From"] = from_addr
     msg["To"] = ", ".join(to_addrs)
     msg["Date"] = formatdate(localtime=True)
     msg["Subject"] = subject
     msg.attach(MIMEText(body, "plain", "utf-8"))
-    smtp = smtplib.SMTP(smtp_server)
+
+    # Connect to server
+    if 'ssl' in smtp and smtp['ssl']:
+        smtp = smtplib.SMTP_SSL(smtp['host'])
+    else:
+        smtp = smtplib.SMTP(smtp['host'])
+    if 'user' in smtp and smtp['user']:
```

Comment:
same, better to use dict.get with default value

Reply:
default value should be '' or null here?

Reply:
@arnaud-morvan it's not the same, and it just should be: 
```
if 'user' in smtp:
```